test/test_x509.py

```diff
@@ -44,6 +45,23 @@ def test_submit_claim_x5c(
     # Sign and submit a dummy claim using our new identity
     claims = crypto.sign_json_claimset(identity, {"foo": "bar"})
     receipt = client.submit_claim(claims).receipt
+    # check if the header struct contains mrenclave header
+    assert "enclave_measurement" in receipt.phdr
+    env_platform = os.environ.get("PLATFORM")
+    actual_measurement = receipt.phdr["enclave_measurement"]
+    expected_virtual_measurement = (
+        "0000000000000000000000000000000000000000000000000000000000000000"
+    )
+    if env_platform == "virtual":
+        assert actual_measurement == expected_virtual_measurement
+    elif env_platform == "sgx":
+        assert (
+            len(actual_measurement) == 64
+            and actual_measurement != expected_virtual_measurement
+        )
```

Comment:
A simpler way here would be to get the `mrenclave` from the running CCF network using the `/node/quotes/self` endpoint and compare that this matches with the mrenclave included in the receipt.

Reply:
Yes and no, because in a test you want to understand what value is expected.